File: javacode/src/main/java/com/federicogualdi/shipmenthub/api/rest/dto/OrderDto.java
package com.federicogualdi.shipmenthub.api.rest.dto;

import java.util.List;

public class OrderDto {

    public int id;
    public DepotDto depot;
    public SupplierDto supplier;
    public List<PackageDto> packages;

    @Override
    public String toString() {
        return "OrderDto{" + "id=" + id + ", depot=" + depot + ", supplier=" + supplier + ", packages=" + packages + '}';
    }
}

File: backoffice/src/api/rest/dto/OrderDto.ts
import { DepotDto } from './DepotDto';
import { PackageDto } from './PackageDto';
import { SupplierDto } from './SupplierDto';

export interface OrderDto {
  id: number;
  depot: DepotDto;
  supplier: SupplierDto;
  packages: PackageDto[];
}
